chapters/syntax.tex

```diff
@@ -26,7 +26,7 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
          "\a" | "\b" | "\f" | "\n" | "\r" | "\t" | "\v"
 DIGIT = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
 UNSIGNED-INTEGER = DIGIT { DIGIT }
-UNSIGNED-NUMBER = UNSIGNED-INTEGER  "." [ UNSIGNED-INTEGER ]
+REAL-UNSIGNED-NUMBER = UNSIGNED-INTEGER  "." [ UNSIGNED-INTEGER ]
```

Comment:
Why don't we call it `UNSIGNED-REAL`?

Reply:
Good idea! For some reason the previous discussion found the name REAL-UNSIGNED-NUMBER, I couldn't find any discussion so I don't see a problem with changing to a shorter and clearer name.